relay_handler.js

```diff
@@ -34,6 +34,36 @@ function RelayHandler(channel, circuits) {
 
 RelayHandler.prototype.type = 'tchannel.relay-handler';
 
+RelayHandler.prototype.handleLazily = function handleLazily(conn, reqFrame) {
+    var self = this;
+
+    // TODO: provide a by-service-name config hook?
+
+    var rereq = new LazyRelayInReq(conn, reqFrame);
+    var err = rereq.initRead();
+    if (err) {
+        rereq.onError(err);
+        return true;
+    }
+
+    rereq.peer = self.channel.peers.choosePeer(null);
+    if (!rereq.peer) {
+        rereq.onError(errors.NoPeerAvailable());
```

Comment:
it already was, since onError logs, but was a too-different log from the old case for this, plus creating an error isn't necessary here, so will fix to just send an error frame and warn log manually.